digiroad2-viite/src/main/scala/fi/liikennevirasto/viite/dao/LinearLocationDAO.scala

```diff
@@ -119,7 +119,7 @@ class LinearLocationDAO {
   val dateFormatter: DateTimeFormatter = ISODateTimeFormat.basicDate()
 
   val selectFromLinearLocation =
-    """
+    s"""
```

Comment:
This should not be in this PR, right - At least it will create unnecessary conflicts with changes in VIITE-2344.